config-templates/netapp/netapp-ontap-nas/changelog.json

```diff
@@ -3,18 +3,18 @@
         "24.02": {
             "status": "supported",
             "latest-revision": {
-                "revision": "3",
-                "release-date": "01 October 2024",
-                "cves": "CVE-2024-34158, CVE-2024-34155, CVE-2024-34156",
+                "revision": "4",
+                "release-date": "28 April 2025",
+                "cves": "CVE-2025-0395",
                 "golang": "",
```

Comment:
I think we should add golang version since its base image is golang and its showing VA for Golang.